doc/doxygen/headers/glossary.h

```diff
@@ -1476,7 +1476,7 @@ Article{JK10,
  *
  *
  * <dt class="glossary">@anchor workstream_paper <b>%WorkStream paper</b></dt>
- * <dd>The "%WorkStream paper" is a paper by B. Turcksin, M. Kronbichler and W. Bangerth
+ * <dd>The "WorkStream paper" is a paper by B. Turcksin, M. Kronbichler and W. Bangerth
```

Comment:
No, see [1]. Doxygen is not creating a link even with this change, probably because of the quotes.

[1] https://www.dealii.org/developer/doxygen/deal.II/DEALGlossary.html#workstream_paper